### PR TITLE
feat: added modal

### PR DESCRIPTION
**Pull request by hiteshverma27**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (rohanmathur91) [inline .vscode/settings.json:3]: Put this in .gitignore or delete it.

File: Styles/modal.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.modal{
+    border-radius: 0.5rem;
+    outline: 2px solid lightgray;
+    background-color: white;
```
Comment (rohanmathur91) [inline Styles/modal.css:4]: have variables for these colors.

File: Styles/modal.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    bottom: 0;
+    position: relative;
+}
+.modal-center{
+    border-radius: 0.5rem;
+    outline: 2px solid lightgray;
+    background-color: white;
```
Comment (rohanmathur91) [inline Styles/modal.css:19]: same as above

**Review by rohanmathur91 — APPROVED** [review]
Suggested few changes, rest looks good.

File: Styles/modal.css (hunk 2 above)
Comment (hiteshverma27) [reply Styles/modal.css:4]: sure, will do that

**Review by yogesh- — COMMENTED** [review]
LGTM. Work on the changes already suggested.

**Pull request merged**